tests/integration_tests/testsuite/ovc_tests/a_basic/cloudspaces_tests.py

```diff
@@ -138,6 +148,259 @@ def test003_create_cloudspace_with_different_limitaions(self, type, factor):
             self.assertEqual(cloudspace['resourceLimits']['CU_I'], CU_I)
             self.assertEqual(cloudspace['resourceLimits']['CU_M'], CU_M)
             self.assertEqual(cloudspace['resourceLimits']['CU_NP'], CU_NP)
-            self.assertIn('gig_qa_1@itsyouonline',[user['userGroupId'] for user in cloudspace['acl']])
+            self.assertIn('gig_qa_1@itsyouonline', [user['userGroupId'] for user in cloudspace['acl']])
+
+        self.log('%s ENDED' % self._testID)
+
+
+class CloudspaceActions(OVC_BaseTest):
```

Comment:
for this class please make sure the tear down for the parent class is not called till all the tests finished